synapse/models/orgs.py

```diff
@@ -1,9 +1,197 @@
 from synapse.common import guid
 
-import synapse.lib.tufo as s_tufo
 import synapse.lib.module as s_module
 
-class OuMod(s_module.CoreModule):
+
+class OuModule(s_module.CoreModule):
+    def getModelDefs(self):
+        modl = {
+            'types': (
+                ('ou:sic', ('str', {'regex': r'^[0-9]{4}$'}), {
+                    'doc': 'The four digit Standard Industrial Classification Code.',
+                    'ex': '0111',
+                }),
+                ('ou:naics', ('str', {'regex': r'^[1-9][0-9]{4}[0-9]?$'}), {
+                    'doc': 'The five or six digit North American Industry Classification System code.',
+                    'ex': '541715',
+                }),
+                ('ou:org', ('guid', {}), {
+                    'doc': 'A GUID for a human organization such as a company or military unit',
+                    'ex': '',  # FIXME Add example
+                }),
+                ('ou:alias', ('str', {'lower': True, 'regex': r'^[0-9a-z]+$'}), {
+                    'doc': 'An alias for the org GUID',  # FIXME: How is this actually an alias?
+                    'ex': 'vertexproject',
+                }),
+                ('ou:hasalias', ('comp', {'fields': (('org', 'ou:org'), ('alias', 'ou:alias'))}), {
```

Comment:
@therealsilence - do we need to have `ou:hasalias` and `ou:org:has`? Seems redundant

Reply:
an org cannot control the aliases it has